src/new-client/src/components/DraggableComponent/DraggableComponent.tsx

```diff
@@ -0,0 +1,87 @@
+import styled from "styled-components"
+import { RefObject, useContext, useState } from "react"
+import { TearOutContext } from "@/App/TearOutSection/tearOutContext"
+import { tearOutSection, Section } from "@/App/TearOutSection/state"
+import { canDrag } from "@/components/DraggableComponent/canDrag"
+import { tearOut } from "@/App/LiveRates/Tile/TearOut/state"
+
+interface DraggableComponentProps {
+  children: JSX.Element[] | JSX.Element
+  section: Section
+  isTile?: boolean
+  ref?: RefObject<HTMLDivElement>
+  isTornOut?: boolean
```

Comment:
I'm not sure we need this. Classic doesn't let you drag things to pop them back into the application, I think this is intentional.
Perhaps we should have a 'disabled' prop here (which we would use `isTornOut` from the parent component to determine if it's disabled or not). 
EG:
`draggable={canDrag && !disabled}`

Reply:
Partially fixed, the only thing I'm not quite happy with the disabled prop, when we use the `draggableTearOut` for the sections, we don't know if it's torn out or not there (hence we would have to import the context function, generate the context and pass the prop), since we would have to do that x3 times, I don't know if it's preferable to contain that logic within  the `DraggableComponent`.

So, in this case we determine if its a Tile, in that case we can pass the prop without issues since the parent has that value, with the sections, in order to pass the prop we would have to drill it from the grandparent (with the components being lazyloaded in between) so we just generate the contest in `DraggableTearOut` and use it from there.

`const disabled = props.isTile ? props.disabled! : tearOutContext.isTornOut`